skins/laika/src/pages/donation_form.ts

```diff
@@ -32,7 +34,10 @@ interface DonationFormModel {
 }
 
 const pageData = new PageDataInitializer<DonationFormModel>( '#app' );
-const store = createStore( [ createFeatureTogglePlugin( pageData.selectedBuckets ) ] );
+const store = createStore( [
+	createFeatureTogglePlugin( pageData.selectedBuckets ),
```

Comment:
Keep in mind that #1741 removes the feature toggle Vuex plugin. If you merge this PR quickly, someone else has to deal with the merge conflicts 😈

Reply:
I know you'd never do that to me!